### DISCUSSION TITLE
Ukrainian sentences that should be supported

### DISCUSSION THREAD

**Discussion by skynetua** [OHF-Voice/intents#1718] [Supported Sentences]
Thread to discuss sentences that not supported yet.

**Comment by zm2GH** [comment]
Як запустити автоматизацію? 
Пуск пилососа та повернення його на базу теж не знайшов.

**Reply by skynetua** [reply]
Автоматизації можна через кастомний тригер, деталі тут https://community.home-assistant.io/t/how-trigger-an-automation-with-assist/538969

В січневому апдейті можна буде [запустити скрипт](https://github.com/home-assistant/intents/pull/1733), сказавши 'Почни прибирання', за умови, що є скрипт з назвою прибирання.

**Reply by zm2GH** [reply]
Насправді я і зараз можу запустити пилосос та запустити автоматизацію через створення бінарних помічників, які додати тригером, а їх "включати/вимикати". Але ж хочеться природньої мови. Тому й треба "запусти, припини, почни, скажи, ..." для псевдонімів "трансляцію, прибирання, заряджання, нагрівання, ..."

**Reply by skynetua** [reply]
#1978 Додали нативні інтенти для пуску пилососа та повернення на базу. Будуть доступні з версії 2024.3
